tasks/updateconstants.js

```diff
@@ -50,6 +51,12 @@ const extraAttribKeys = [
 
 const now = Number(new Date());
 
+aghs_desc_urls = [];
+for (i=1; i <= 200; i++)
+{
+  aghs_desc_urls.push("http://www.dota2.com/datafeed/herodata?language=english&hero_id=" + i);
+}
```

Comment:
This might able to be done better. I'd rather not hardcode it to 200, but for now it works. Definitely open to suggestions.

Reply:
Can you use the hero list from heroes.json? Or fetch from an endpoint?

Reply:
I've tried fetching from an endpoint for far too long. LOL. It really just rattles this code because this script is setup to asynchronously get data from multiple endpoints based on the URLs you should of had setup previously. I'm not sure how to make everything stop and wait for me to fill up this array of URLs. I'm sure there's a way, I really am just not good at web development!

I think I'll have to just use heroes.json. I'm not a huge fan of creating the dependency of something that is created by this script asynchronously, but considering I don't have to request anything from an endpoint, I should be able to do this completely serially.

Reply:
yeah you can just require heroes.json from the top and it should work. It won't pick up a new hero if it's added though.

Or you can add https://raw.githubusercontent.com/dotabuff/d2vpkr/master/dota/scripts/npc/npc_heroes.json to your list of urls, and then treat that first response specially.

Reply:
using heroes.json now.